src/blog/2022/09/flowforge-010-released.md

```diff
@@ -0,0 +1,78 @@
+---
+title: FlowForge 0.10 released
+subtitle: Secure HTTP end points, Read-only users and Static outbound IPs
+description: Secure Node-RED HTTP end points, Read-only users and Static outbound IPs
+date: 2022-09-30 12:00:00.0
+authors: ["rob-marcer"]
+video: d23Pmyc0k7I
+---
+
+Secure your HTTP endpoints, create read-only users in your teams and use our static IP address for outbound traffic
+
+<!--more-->
+
+Keep reading for the details of what's in this release or you can watch our 1 minute roundup video of the new release above. 
+
+We're pleased to announce version 0.10 is now available. The next release of the FlowForge application containing new features, a number of improvements, and bug fixes. Keep reading for a promotion code to get your first month free on FlowForge Cloud. 
+
+## Features
+[Secure HTTP Endpoints](https://github.com/flowforge/flowforge/issues/578)
+We've added the ability for you to secure your HTTP endpoints. You can now control who can access Dashboards or API endpoints you create in FlowForge.
+
+[Read-only Users](https://github.com/flowforge/flowforge/issues/657)
+We've added a new user role for Read-only access. This will allow users to login to your FlowForge project and view the Node-RED flows including the debug output without them being able to edit anything.
+
+[Static Outbound IP Addresses](https://github.com/flowforge/CloudProject/issues/59)
+We've updated FlowForge Cloud so that all outbound traffic from your projects now comes from a single IP address. When trying to access a remote resource such as a database it is often a requirement for the IP address the traffic comes from to be known remain unchanged. 
```

Comment:
```suggestion
We've updated FlowForge Cloud so that all outbound traffic from your projects now comes from a single IP address. When trying to access a remote resource such as a database it is often a requirement for the IP address the traffic comes from to be fixed. 
```